include/yokozuna.hrl

```diff
@@ -258,6 +258,9 @@
             app_helper:get_env(riak_kv, anti_entropy_build_limit, {1, 3600000})
         )).
 
+-define(YZ_ENTROPY_LOCK_TIMEOUT,
+    app_helper:get_env(?YZ_APP_NAME, anti_entropy_lock_timeout, 10000)).
```

Comment:
@fadushin and I have liked using macros in defaults like this. I know it's not *that* important, but I think it's easier to look for it going forward.